web/src/routes/user/lists/+page.server.ts

```diff
@@ -205,7 +205,7 @@ export const actions = {
     const tmdbId = data.get('id') as string;
 
     try {
-      const response = await fetch(`${API}/movielists/${listId}`, {
+      const response = await fetch(`${API}/movies/${listId}/update`, {
```

Comment:
Adjust this to the changes [here](https://github.com/funnls/cinesync/pull/97/files#r1610783360).